docs/source/dev_guide/algorithms/svm/support-vector-machine-classifier.rst

```diff
@@ -230,36 +232,49 @@ Examples
 
 .. tabs::
 
-  .. tab:: DPC++
+  .. tab:: oneAPI DPC++
+
+    Batch Processing:
+    
+    - :ref:`dpc_svm_two_class_thunder_dense_batch.cpp`
+
+  .. tab:: oneAPI C++
 
     Batch Processing:
     
-    -  :ref:`svm_two_class_thunder_dense_batch.cpp`
+    - :ref:`cpp_svm_two_class_smo_dense_batch.cpp`
```

Comment:
oneAPI C++ have 2 examples for SVM (SMO & thunder method)
```
:ref:`cpp_svm_two_class_thunder_dense_batch.cpp
:ref:`cpp_svm_two_class_smo_dense_batch.cpp
```

Reply:
fixed